tensor/api_arith.go

```diff
@@ -459,3 +459,20 @@ func Outer(a, b Tensor, opts ...FuncOpt) (retVal Tensor, err error) {
 	}
 	panic("Unreachable")
 }
+
+// Contract performs a contraction of given tensors along given axes
+func Contract(aAxes, bAxes []int, a, b Tensor) (retVal Tensor, err error) {
```

Comment:
can the API of this change to 

`func Contract(a, b Tensor, axesA, axesB []int) (retVal Tensor, err error)`?